Header header
float32 det_percentage
float32 det_fps
float32 secs_till_last